genet/utils/google_directions.py

```diff
@@ -67,18 +69,43 @@ def read_saved_api_results(file_path):
     return api_requests
 
 
-def make_request(origin_attributes, destination_attributes, key, traffic):
+def make_request(origin_attributes, destination_attributes, key, departure_time, traffic_model):
     base_url = 'https://maps.googleapis.com/maps/api/directions/json'
-    params = {
-        'origin': '{},{}'.format(origin_attributes['lat'], origin_attributes['lon']),
-        'destination': '{},{}'.format(destination_attributes['lat'], destination_attributes['lon']),
-        'key': key
-    }
-    if traffic:
+    params = {'origin': '{},{}'.format(origin_attributes['lat'], origin_attributes['lon']),
+              'destination': '{},{}'.format(destination_attributes['lat'], destination_attributes['lon']), 'key': key,
+              'traffic_model': traffic_model}
+    current_unix_time = time.time()
```

Comment:
but we are now using a variable for the `end_of_unix_time` so I guess for consistency this should be a variable too? it's easier to understand too

Reply:
Well, the unix time max value is a constant - it never changes. We should use uppercase for constants, by the way, so that should become `END_OF_UNIX_TIME`. And you'd usually declare them at the module level, rather than inside a function. Unlike Java, you cannot programmatically make the value final, otherwise we'd do that too.

This one is actually a variable, so I wouldn't want to treat it as a constant. And I think the difference in clarity and intent between `current_unix_time` and `time.time()` is much smaller than the one between `2147483646` and `END_OF_UNIX_TIME`. But it's not a big deal either way.